.github/workflows/tests-datapath-verifier.yaml

```diff
@@ -84,31 +84,41 @@ jobs:
         include:
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: 'rhel8.6-20250812.093650'
+            kernel-name: 'rhel8.6'
             ci-kernel: '510'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: '5.10-20250923.075842'
+            kernel-name: '5.10'
             ci-kernel: '510'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: '5.15-20250923.075842'
+            kernel-name: '5.15'
             ci-kernel: '510'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: '6.1-20250923.075842'
+            kernel-name: '6.1'
             ci-kernel: '61'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: '6.6-20250923.075842'
+            kernel-name: '6.6'
             ci-kernel: '61'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: '6.12-20250923.075842'
+            kernel-name: '6.12'
             ci-kernel: '61'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: 'bpf-next-20250923.075842'
+            kernel-name: 'bpf-next'
             ci-kernel: 'netnext'
     timeout-minutes: 60
     steps:
-      - name: Collect Workflow Telemetry
-        uses: catchpoint/workflow-telemetry-action@94c3c3d9567a0205de6da68a76c428ce4e769af1 # v2.0.0
+      - name: Checkout merge branch
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        if: ${{ github.event_name == 'workflow_dispatch' }}
         with:
-          comment_on_pr: false
+          ref: ${{ github.sha }}
```

Comment:
`${{ github.sha }}` does not work as reliable SHA for the base branch. That is due to the context in which the workflow_dispatch is called when a PR is from a branch on the cilium/cilium repo. So this only yields valid results on PRs from forks.

I have a PR underway to add the base SHA to Ariana, one that is live I can fix the workflow, we should hold off on merging until then.